### ISSUE TITLE
FAQ

### ISSUE THREAD

**Issue by agriffard** [OrchardCMS/OrchardCore#4638]
In addition to Glossary(#4572), and if we decide to add a FAQ page in the documentation: 
What would be the most important questions to answer to?

We will try not be to specific and too high level in this page, so please, ask questions as if you were a beginner discovering OC (as we all have been) and that wanted to understand the main concepts.

May be also explain what is does well and why you would choose it according to your needs or what it is not suited for.

**Comment by agriffard** [comment]
Some of the most frequent questions for a beginner that creates a new module:

Q1: Why the module I created does not appear in the admin?
or
Q2: Why the admin menu I created for this feature does not appear?

A1: Did you reference the project in your application? 
and
A2: Did you enable the feature in the Features admin page?
and
A3: Did you add the AdminMenu in the Startup.cs? 
=> Oops, ok it works now.
https://orchardcore.readthedocs.io/en/dev/docs/guides/add-admin-menu/#adminmenucs
